### PR TITLE
[9.0][ADD] request_auditor

### PR DESCRIPTION
**Pull request by andreparames**
This is the same module as in #1160, just with a couple of renames (`odoo` → `openerp`).

### CODE REVIEW

File: request_auditor/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# © 2018 ACSONE SA/NV <https://acsone.eu/>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Audit",
+    "version": "10.0.1.0.0",
```
Comment (ThomasBinsfeld) [inline request_auditor/__openerp__.py]: @andreparames 10.0.1.0.0 -> 9.0.1.0.0

**Review by ThomasBinsfeld — CHANGES_REQUESTED** [review]
Just change the version in the manifest file and regenerate the setup to be compatible with 9.0 (setup must be in odoo_addons in 9.0, not in odoo/addons). That can be done with setuptools-odoo-make-default -d

**Comment by andreparames** [comment]
@ThomasBinsfeld thanks.

**Review by ThomasBinsfeld — APPROVED** [review]

**Comment by sbidoul** [comment]
cc/ @lmignon

**Review by lmignon — APPROVED** [review]
As for #1160 :+1:

**Review by ThomasBinsfeld — APPROVED** [review]
Code and functional :+1:

**Pull request closed without merging**